verification/curator-service/api/openapi/openapi.yaml

```diff
@@ -522,9 +689,27 @@ components:
       items:
         $ref: "#/components/schemas/Geocode"
     NewCase:
-      description: The subset of "#/components/schemas/Case" required to create the entity
-      type: object
-      # TODO: Define NewCase object.
+      description: >
+        A "#/components/schemas/Case" with additional required fields for
+        newly-created cases.
+      allOf:
+        - $ref: "#/components/schemas/Case"
+      properties:
+        caseReference:
+          required:
+            - sourceId
+            - sourceUrl
+        location:
+          description: Note that either `geometry` or `query` is required.
```

Comment:
Isn't it `query || (geometry && country && geoResolution)`?

Reply:
yes, thanks!